lib/api/follow.js

```diff
@@ -14,30 +14,30 @@ const database = require('../database');
  *
  * @returns {object} - All items with attributes and the pagination for the items
  */
-const follow = (query, rows, types, total, organize) => {
-  return database.transaction(trx => {
-    const items = rows.map(row => {
-      const type = types.find(model => {
-        return model.id === row['type-slug'];
-      });
-
-      if (typeof type === 'undefined') {
-        return new Promise(res => {
-          res({});
-        });
-      }
+const follow = (query, rows, types, total, organize, trx) => {
+  const db = trx || database;
```

Comment:
This needs a little work. If `trx` isn't passed in, it needs to create a transaction.